ui/src/app/app.module.ts

```diff
@@ -10,15 +11,16 @@ import { LoginComponent }       from './login/login.component';
 import { MastheadComponent }    from './masthead/masthead.component';
 
 const routes: Routes = [
-    { path: 'dashboard', component: DashboardComponent },
+    { path: '', component: DashboardComponent },
     { path: 'login',     component: LoginComponent }
 ];
 
 @NgModule({
     imports: [
         BrowserModule,
         FormsModule,
-        RouterModule.forRoot(routes, { useHash: true })
+        RouterModule.forRoot(routes),
```

Comment:
How will removing useHash be used in conjunction with the API? Will routes need to be added just to Go and not Angular?